picard/script.py

```diff
@@ -475,57 +492,78 @@ def func_trim(parser, text, char=None):
         return text.strip()
 
 
-def func_add(parser, x, y):
-    """Add ``y`` to ``x``."""
+def func_add(parser, x, y, *args):
+    """Adds ``y`` to ``x``.
+       Can be used with arbitrary number of arguements. The result is
+       $add($add(x,y)...)
+    """
     try:
-        return str(int(x) + int(y))
+        return _compute('add', locals())
```

Comment:
I wanted 2 variables to be necessary.

I can just check for the length of args and raise an error though.

Any thoughts on how else it can be implemented?